{"uid":"a035deaf500bfd6c","name":"Проверка создания пользователя","fullName":"tests.test_ui.test_first#test_positive_registration","historyId":"43c63fead18b01a08ec1d8de921e11ef","time":{"start":1760785729563,"stop":1760785733116,"duration":3553},"description":"\nЭтот тест проверяет:\n- Регистрацию пользователя с позитивным сценарием\n- Проверка успеха прохождения регистрации \nИЛИ\n- Проверка предупреждения, что пользователь уже создан с таким email\n","descriptionHtml":"<p>Этот тест проверяет:</p>\n<ul>\n<li>Регистрацию пользователя с позитивным сценарием</li>\n<li>Проверка успеха прохождения регистрации<br />\nИЛИ</li>\n<li>Проверка предупреждения, что пользователь уже создан с таким email</li>\n</ul>\n","status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=141.0.7390.107)\nStacktrace:\n#0 0x5595a2fb067a <unknown>\n#1 0x5595a2a2f566 <unknown>\n#2 0x5595a2a26470 <unknown>\n#3 0x5595a2a17926 <unknown>\n#4 0x5595a2a1972b <unknown>\n#5 0x5595a2a17eb1 <unknown>\n#6 0x5595a2a17650 <unknown>\n#7 0x5595a2a17302 <unknown>\n#8 0x5595a2a15142 <unknown>\n#9 0x5595a2a1592a <unknown>\n#10 0x5595a2a32a99 <unknown>\n#11 0x5595a2acdee5 <unknown>\n#12 0x5595a2aa7632 <unknown>\n#13 0x5595a2acd328 <unknown>\n#14 0x5595a2aa7403 <unknown>\n#15 0x5595a2a73b02 <unknown>\n#16 0x5595a2a747c1 <unknown>\n#17 0x5595a2f74448 <unknown>\n#18 0x5595a2f782af <unknown>\n#19 0x5595a2f5b8d9 <unknown>\n#20 0x5595a2f78e55 <unknown>\n#21 0x5595a2f4113f <unknown>\n#22 0x5595a2f9d4b8 <unknown>\n#23 0x5595a2f9d693 <unknown>\n#24 0x5595a2faf613 <unknown>\n#25 0x7f9d66008b7b <unknown>","statusTrace":"browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"bf5505b75050c7b90d5939c7cb5ad46a\")>\nbase_url = 'http://31.59.174.108'\nwait = <selenium.webdriver.support.wait.WebDriverWait (session=\"bf5505b75050c7b90d5939c7cb5ad46a\")>\nregistration_data = {'email': 'test_user123@mail.ru', 'name': 'User25', 'password': 'qwertyui'}\n\n    @allure.epic(\"Аутентификация и авторизация\")\n    @allure.feature(\"Регистрация пользователей\")\n    @allure.suite(\"Смоук тесты\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    @allure.tag(\"smoke\", \"ui\")\n    @allure.label(\"owner\", \"QA - evgeny_gord\")\n    @allure.id(\"TEST-123\")\n    @allure.title(\"Проверка создания пользователя\")\n    @allure.description(\"\"\"\n    Этот тест проверяет:\n    - Регистрацию пользователя с позитивным сценарием\n    - Проверка успеха прохождения регистрации\n    ИЛИ\n    - Проверка предупреждения, что пользователь уже создан с таким email\n    \"\"\")\n    @allure.link(\"https://jira.example.com/TEST-123\", name=\"JIRA Task\")\n    @allure.issue(\"BUG-456\", \"https://bugtracker.com/BUG-456\")\n    @allure.testcase(\"TC-789\", \"https://testcase.com/TC-789\")\n    def test_positive_registration(browser, base_url, wait, registration_data):\n        # driver = webdriver.Chrome(service=Service(executable_path='C:/chromedriver.exe'))\n        # driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()))\n        # driver.get(url)\n        with allure.step('Переход на страницу Регистрации'):\n>           browser.get(f\"{base_url}/sign_up\")\n\ntests/test_ui/test_first.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:483: in get\n    self.execute(Command.GET, {\"url\": url})\n/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:458: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f59f72de600>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5595a2f9d693 \\\\u003Cunknown>\\\\n#24 0x5595a2faf613 \\\\u003Cunknown>\\\\n#25 0x7f9d66008b7b \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\") if isinstance(message, dict) else None\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=141.0.7390.107)\nE       Stacktrace:\nE       #0 0x5595a2fb067a <unknown>\nE       #1 0x5595a2a2f566 <unknown>\nE       #2 0x5595a2a26470 <unknown>\nE       #3 0x5595a2a17926 <unknown>\nE       #4 0x5595a2a1972b <unknown>\nE       #5 0x5595a2a17eb1 <unknown>\nE       #6 0x5595a2a17650 <unknown>\nE       #7 0x5595a2a17302 <unknown>\nE       #8 0x5595a2a15142 <unknown>\nE       #9 0x5595a2a1592a <unknown>\nE       #10 0x5595a2a32a99 <unknown>\nE       #11 0x5595a2acdee5 <unknown>\nE       #12 0x5595a2aa7632 <unknown>\nE       #13 0x5595a2acd328 <unknown>\nE       #14 0x5595a2aa7403 <unknown>\nE       #15 0x5595a2a73b02 <unknown>\nE       #16 0x5595a2a747c1 <unknown>\nE       #17 0x5595a2f74448 <unknown>\nE       #18 0x5595a2f782af <unknown>\nE       #19 0x5595a2f5b8d9 <unknown>\nE       #20 0x5595a2f78e55 <unknown>\nE       #21 0x5595a2f4113f <unknown>\nE       #22 0x5595a2f9d4b8 <unknown>\nE       #23 0x5595a2f9d693 <unknown>\nE       #24 0x5595a2faf613 <unknown>\nE       #25 0x7f9d66008b7b <unknown>\n\n/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1760785670477,"stop":1760785670646,"duration":169},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"browser","time":{"start":1760785685634,"stop":1760785729557,"duration":43923},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"base_url","time":{"start":1760785729559,"stop":1760785729561,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"wait","time":{"start":1760785729561,"stop":1760785729561,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"registration_data","time":{"start":1760785729562,"stop":1760785729562,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0}],"testStage":{"description":"\nЭтот тест проверяет:\n- Регистрацию пользователя с позитивным сценарием\n- Проверка успеха прохождения регистрации \nИЛИ\n- Проверка предупреждения, что пользователь уже создан с таким email\n","status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=141.0.7390.107)\nStacktrace:\n#0 0x5595a2fb067a <unknown>\n#1 0x5595a2a2f566 <unknown>\n#2 0x5595a2a26470 <unknown>\n#3 0x5595a2a17926 <unknown>\n#4 0x5595a2a1972b <unknown>\n#5 0x5595a2a17eb1 <unknown>\n#6 0x5595a2a17650 <unknown>\n#7 0x5595a2a17302 <unknown>\n#8 0x5595a2a15142 <unknown>\n#9 0x5595a2a1592a <unknown>\n#10 0x5595a2a32a99 <unknown>\n#11 0x5595a2acdee5 <unknown>\n#12 0x5595a2aa7632 <unknown>\n#13 0x5595a2acd328 <unknown>\n#14 0x5595a2aa7403 <unknown>\n#15 0x5595a2a73b02 <unknown>\n#16 0x5595a2a747c1 <unknown>\n#17 0x5595a2f74448 <unknown>\n#18 0x5595a2f782af <unknown>\n#19 0x5595a2f5b8d9 <unknown>\n#20 0x5595a2f78e55 <unknown>\n#21 0x5595a2f4113f <unknown>\n#22 0x5595a2f9d4b8 <unknown>\n#23 0x5595a2f9d693 <unknown>\n#24 0x5595a2faf613 <unknown>\n#25 0x7f9d66008b7b <unknown>","statusTrace":"browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"bf5505b75050c7b90d5939c7cb5ad46a\")>\nbase_url = 'http://31.59.174.108'\nwait = <selenium.webdriver.support.wait.WebDriverWait (session=\"bf5505b75050c7b90d5939c7cb5ad46a\")>\nregistration_data = {'email': 'test_user123@mail.ru', 'name': 'User25', 'password': 'qwertyui'}\n\n    @allure.epic(\"Аутентификация и авторизация\")\n    @allure.feature(\"Регистрация пользователей\")\n    @allure.suite(\"Смоук тесты\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    @allure.tag(\"smoke\", \"ui\")\n    @allure.label(\"owner\", \"QA - evgeny_gord\")\n    @allure.id(\"TEST-123\")\n    @allure.title(\"Проверка создания пользователя\")\n    @allure.description(\"\"\"\n    Этот тест проверяет:\n    - Регистрацию пользователя с позитивным сценарием\n    - Проверка успеха прохождения регистрации\n    ИЛИ\n    - Проверка предупреждения, что пользователь уже создан с таким email\n    \"\"\")\n    @allure.link(\"https://jira.example.com/TEST-123\", name=\"JIRA Task\")\n    @allure.issue(\"BUG-456\", \"https://bugtracker.com/BUG-456\")\n    @allure.testcase(\"TC-789\", \"https://testcase.com/TC-789\")\n    def test_positive_registration(browser, base_url, wait, registration_data):\n        # driver = webdriver.Chrome(service=Service(executable_path='C:/chromedriver.exe'))\n        # driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()))\n        # driver.get(url)\n        with allure.step('Переход на страницу Регистрации'):\n>           browser.get(f\"{base_url}/sign_up\")\n\ntests/test_ui/test_first.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:483: in get\n    self.execute(Command.GET, {\"url\": url})\n/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:458: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f59f72de600>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5595a2f9d693 \\\\u003Cunknown>\\\\n#24 0x5595a2faf613 \\\\u003Cunknown>\\\\n#25 0x7f9d66008b7b \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\") if isinstance(message, dict) else None\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=141.0.7390.107)\nE       Stacktrace:\nE       #0 0x5595a2fb067a <unknown>\nE       #1 0x5595a2a2f566 <unknown>\nE       #2 0x5595a2a26470 <unknown>\nE       #3 0x5595a2a17926 <unknown>\nE       #4 0x5595a2a1972b <unknown>\nE       #5 0x5595a2a17eb1 <unknown>\nE       #6 0x5595a2a17650 <unknown>\nE       #7 0x5595a2a17302 <unknown>\nE       #8 0x5595a2a15142 <unknown>\nE       #9 0x5595a2a1592a <unknown>\nE       #10 0x5595a2a32a99 <unknown>\nE       #11 0x5595a2acdee5 <unknown>\nE       #12 0x5595a2aa7632 <unknown>\nE       #13 0x5595a2acd328 <unknown>\nE       #14 0x5595a2aa7403 <unknown>\nE       #15 0x5595a2a73b02 <unknown>\nE       #16 0x5595a2a747c1 <unknown>\nE       #17 0x5595a2f74448 <unknown>\nE       #18 0x5595a2f782af <unknown>\nE       #19 0x5595a2f5b8d9 <unknown>\nE       #20 0x5595a2f78e55 <unknown>\nE       #21 0x5595a2f4113f <unknown>\nE       #22 0x5595a2f9d4b8 <unknown>\nE       #23 0x5595a2f9d693 <unknown>\nE       #24 0x5595a2faf613 <unknown>\nE       #25 0x7f9d66008b7b <unknown>\n\n/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","steps":[{"name":"Переход на страницу Регистрации","time":{"start":1760785729563,"stop":1760785733099,"duration":3536},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=141.0.7390.107)\nStacktrace:\n#0 0x5595a2fb067a <unknown>\n#1 0x5595a2a2f566 <unknown>\n#2 0x5595a2a26470 <unknown>\n#3 0x5595a2a17926 <unknown>\n#4 0x5595a2a1972b <unknown>\n#5 0x5595a2a17eb1 <unknown>\n#6 0x5595a2a17650 <unknown>\n#7 0x5595a2a17302 <unknown>\n#8 0x5595a2a15142 <unknown>\n#9 0x5595a2a1592a <unknown>\n#10 0x5595a2a32a99 <unknown>\n#11 0x5595a2acdee5 <unknown>\n#12 0x5595a2aa7632 <unknown>\n#13 0x5595a2acd328 <unknown>\n#14 0x5595a2aa7403 <unknown>\n#15 0x5595a2a73b02 <unknown>\n#16 0x5595a2a747c1 <unknown>\n#17 0x5595a2f74448 <unknown>\n#18 0x5595a2f782af <unknown>\n#19 0x5595a2f5b8d9 <unknown>\n#20 0x5595a2f78e55 <unknown>\n#21 0x5595a2f4113f <unknown>\n#22 0x5595a2f9d4b8 <unknown>\n#23 0x5595a2f9d693 <unknown>\n#24 0x5595a2faf613 <unknown>\n#25 0x7f9d66008b7b <unknown>\n\n","statusTrace":"  File \"/app/tests/test_ui/test_first.py\", line 41, in test_positive_registration\n    browser.get(f\"{base_url}/sign_up\")\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 483, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 458, in execute\n    self.error_handler.check_response(response)\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"attachmentStep":false,"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0}],"attachments":[{"uid":"a8f592a61644df02","name":"stdout","source":"a8f592a61644df02.txt","type":"text/plain","size":85}],"parameters":[],"hasContent":true,"attachmentStep":false,"stepsCount":1,"shouldDisplayMessage":true,"attachmentsCount":1},"afterStages":[{"name":"browser::0","time":{"start":1760785733309,"stop":1760785733588,"duration":279},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0}],"labels":[{"name":"tag","value":"smoke"},{"name":"severity","value":"critical"},{"name":"feature","value":"Регистрация пользователей"},{"name":"epic","value":"Аутентификация и авторизация"},{"name":"suite","value":"Смоук тесты"},{"name":"owner","value":"QA - evgeny_gord"},{"name":"as_id","value":"TEST-123"},{"name":"tag","value":"ui"},{"name":"parentSuite","value":"tests.test_ui"},{"name":"host","value":"ecb2b429c773"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_ui.test_first"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[{"name":"https://testcase.com/TC-789","url":"TC-789","type":"tms"},{"name":"https://bugtracker.com/BUG-456","url":"BUG-456","type":"issue"},{"name":"JIRA Task","url":"https://jira.example.com/TEST-123","type":"link"}],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["ui","smoke"]},"source":"a035deaf500bfd6c.json","parameterValues":[]}